src/run.c

```diff
@@ -1,60 +1,64 @@
-#include <errno.h>
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
 #include "jou_compiler.h"
-#include <llvm-c/Core.h>
+#include <llvm-c/ExecutionEngine.h>
+#include <llvm-c/Transforms/PassManagerBuilder.h>
 
+static void optimize(LLVMModuleRef module, int level)
+{
+    assert(0 <= level && level <= 3);
 
-static char TempDir[50];
+    LLVMPassManagerRef pm = LLVMCreatePassManager();
 
-static void cleanup()
-{
-    char command[200];
-    sprintf(command, "rm -rf '%s'", TempDir);
-    system(command);
+    LLVMPassManagerBuilderRef pmbuilder = LLVMPassManagerBuilderCreate();
```

Comment:
```suggestion
    /*
    The default settings should be fine for Jou because they work well for
    C and C++, and Jou is quite similar to C.
    */
    LLVMPassManagerBuilderRef pmbuilder = LLVMPassManagerBuilderCreate();
```